src/plays/download-fetch-response/Readme.md

```diff
@@ -0,0 +1,16 @@
+# download-fetch-response
+
+Save the FETCH API response as a file.
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Implementation Details
+
+To accomplish this, a custom hook for the fetch (GET)API is written along with UseEffect which includes a cleanup function .
+
+## Consideration
+
+Works for all valid get API's(provided token is not mandatory)
```

Comment:
I would suggest we add this as a note in the UI too.

Reply:
Added